bin/publish_helm_chart

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -evuo pipefail
+
+if [ $PUBLISH_HELM_CHART != true ]; then
+    echo "publish flag not set"
+    exit 1
+fi
+
+echo "Installing helm"
+
+curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh --version "${HELM_VERSION}"
+helm init --client-only
+
+echo "Publishing helm chart"
+
+output="$(cd helm; helm package fiaas-skipper)"
+echo $output
+package=`expr "$output" : 'Successfully packaged chart and saved it to: \(.*\)'`
+echo $package
+
+git clone https://github.com/fiaas/helm helm-repo
+mv $package helm-repo/
+cd helm-repo/ && helm repo index . --url https://fiaas.github.io/helm/ && git add . && git commit -a -m "Release fiaas-skipper" && git push https://${GIT_USERNAME}:${GIT_APITOKEN}@github.com/fiaas/helm && echo "Successfully pushed helm chart"
```

Comment:
Since we're setting `errexit`, maybe these commands can be separated by newlines rather than being a long one-liner?

Reply:
sure